src/templates/static.tsx

```diff
@@ -47,7 +47,7 @@ type ExternalImageData = TemplateProps & { externalImage: ExternalImage };
 export const transformProps: TransformProps<ExternalImageData> = async (
   data
 ) => {
-  const url = import.meta.env.YEXT_PUBLIC_EXTERNAL_IMAGE_API_BASE_URL + "/2";
+  const url = "https://jsonplaceholder.typicode.com/photos/2";
```

Comment:
Let's not make this change right now as the documentation references the environment variable here. Let's just use the new syntax with `YEXT_PUBLIC` and when I update the docs, I'll make a new PR to adjust this

Reply:
Messaged on slack about needing to remove this in order to remove .env from source control